rpcs3.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
+# Visual Studio 15
+VisualStudioVersion = 15.0.27004.2002
```

Comment:
Is this version number from Visual Studio 2017?

Reply:
I think so , I've made the changes in VS 2017 .

Reply:
Pretty sure we target VS2015, @Nekotekina ?

Reply:
It's possible to use solution files from VS2015 on VS2017 without altering them. So it seems reasonable to keep VS2015 support because users of VS2017 can build RPCS3 just fine, however the reverse might not be true.